lib/JSON/Validator/Ref.pm

```diff
@@ -0,0 +1,60 @@
+package JSON::Validator::Ref;
```

Comment:
@jberger: This class should fix your issues with the document renderer, since it keeps the original $ref instead of changing it.